### PR TITLE
newspaper titles page

### PR DESCRIPTION
**Pull request by tvanbeek**
added page with list of all newspaper titles and detail pages with previews of the frontpages and metadata

### CODE REVIEW

File: src/router/index.js [hunk 1]
```diff
@@ -110,6 +111,14 @@ export default new Router({
         realm: 'issueviewer',
       },
     },
+    {
+      path: '/titles/:title_uid?',
```
Comment (danieleguido) [inline src/router/index.js]: hehe, why not just "newspapers" :) title is misleading 
https://en.wikipedia.org/wiki/Newspaper

File: src/services/index.js [hunk 2]
```diff
@@ -49,6 +49,7 @@ export const articles = app.service('articles');
 export const issues = app.service('issues');
 export const pages = app.service('pages');
 export const search = app.service('search');
+export const titles = app.service('newspapers');
```
Comment (danieleguido) [inline src/services/index.js]: same here ...

Comment (tvanbeek) [reply src/services/index.js]: fixed in commit b0e3c79

File: src/router/index.js (hunk 1 above)
Comment (tvanbeek) [reply src/router/index.js]: fixed in commit b0e3c79

**Pull request merged**